acceptance/ui/features/applications.feature

```diff
@@ -0,0 +1,161 @@
+@apps
+Feature: Application Management
+  In order to use Control Center
+  As a CC admin user
+  I want to manage applications
+
+  @login-required
+  Scenario: View default applications page
+    When I am on the applications page
+    Then I should see "Application Template"
+      And I should see "Deployment ID"
+      And I should see "Virtual Host Names"
+      And I should see "Internal Services"
+      And I should see "Description"
+      And I should see "Services Map"
+
+  @run @login-required
+  Scenario: View services map
+    When I am on the applications page
+      And I click the Services Map button
+    Then I should see "Internal Services" in the Services Map
+
+  @login-required
+  Scenario: View application template dialog
+    When I am on the applications page
+      And I click the Add-Application Template button
+    Then I should see "Add Application Template"
+      And I should see "Application Template File:"
+      And I should see "No file chosen"
+
+  @login-required
+  Scenario: Add an application
+    When I am on the applications page
+      And I click the Add-Application button
+    Then I should see "Deployment Wizard"
+      And I should see "Select the application to install:"
+    When I select "Zenoss.core"
+      And I click "Next"
+    Then I should see "Select the resource pool to install to:"
+    When I select "default"
+      And I click "Next"
+    Then I should see "Zenoss.core has been configured for resource pool default"
+    When I fill in the Deployment ID field with "Hbase"
+      And I click "Deploy"
+    Then I should see "Pulling images"
+      And I should see "this may take awhile"
+      And I should see "Application deployed"
+
+  @login-required @emptyHostsPage
```

Comment:
The Add Application tests don't work since Cucumber times out before the app is added. I'll remove them for now.